src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -682,4 +682,16 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>Certificate store '{0}' not found. For a list of accepted ways to provide a certificate, please visit https://docs.nuget.org/docs/reference/command-line-reference</value>
     <comment>{0} is the certificate store name</comment>
   </data>
+  <data name="Error_ProjectWithIncorrectDependenciesCount" xml:space="preserve">
+    <value>Project {0} must have exactly {1} package reference(s).</value>
+  </data>
+  <data name="Log_SkippingCompatibiilityCheckOnRidlessGraphForDotnetToolReferenceProject" xml:space="preserve">
+    <value>Skipping compatibility checks of the ridless graph for DotnetToolReference project type. This graph is not used. </value>
+  </data>
+  <data name="Error_InvalidProjectPackageCombo" xml:space="preserve">
+    <value>Invalid project-package combination for {0} {1}. DotnetToolReference project style can only contain references of the DotnetTool type</value>
+  </data>
+  <data name="Error_ToolsPackageWithExtraPackageTypes" xml:space="preserve">
+    <value>Invalid tools package {0} {1}. Tools packages cannot contain more than 1 PackageType.</value>
```

Comment:
nit: as a general rule numbers smaller than 10 should be written out as the word, not as the numeral. So: *more than one PackageType* here